src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -4417,6 +4417,12 @@ public abstract partial class Type : System.Reflection.MemberInfo, System.Reflec
         public static System.Type MakeGenericSignatureType(System.Type genericTypeDefinition, params System.Type[] typeArguments) { throw null; }
         public virtual System.Type MakeGenericType(params System.Type[] typeArguments) { throw null; }
         public virtual System.Type MakePointerType() { throw null; }
+        internal const System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes MemberAnnotations = System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicFields | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.NonPublicFields |
```

Comment:
internal members don't go in our ref file.

You should use the tools to generate this file, which will put all the `|` values inline where they are needed.